src/contracts/libraries/GPv2Trade.sol

```diff
@@ -89,12 +89,12 @@ library GPv2Trade {
             GPv2Signing.Scheme signingScheme
         )
     {
-        if (flags & 0x01 == 0) {
+        if (flags & 0b01 == 0) {
```

Comment:
What's with this change. Doesn't seem like documentation.

Reply:
They have identical values (0x01 == 0b01) and was changed to better reflect the documentation above (since it is written referencing bit positions instead of hex values).